src/MgvOfferMaking.sol

```diff
@@ -399,6 +411,7 @@ contract MgvOfferMaking is MgvHasOffers {
 
       // store offer at the end of the tick
       leaf = leaf.setTickLast(insertionTick, ofrId);
+      // FIXME check whether write necessary: when offer neither first nor last of tick
```

Comment:
How can the offer be not last? In the line before, we set it to be last?

Maybe the idea is not to write if the offer was already last in that tick? Ie an update that doesn't change the tick.

Reply:
The idea was « it’s 2am and I forgot we append to lists » 😄

Reply:
This still needs to be fixed :-)